Frameworks/Foundation/NSDecimalNumber.mm

```diff
@@ -312,34 +350,98 @@ - (NSDecimalNumber*)decimalNumberByRaisingToPower:(NSUInteger)power {
  @Status Stub
  @Notes
 */
-- (NSDecimalNumber*)decimalNumberByMultiplyingByPowerOf10:(short)power {
+- (instancetype)initWithString:(NSString*)numericString locale:(id)locale {
     UNIMPLEMENTED();
     return StubReturn();
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
+*/
+- (NSDecimal)decimalValue {
+    // Read-Only
+    NSDecimal decimal = { 0 };
```

Comment:
All of the fields are set to 0.

---
In reply to: [80787871](https://github.com/Microsoft/WinObjC/pull/1032#discussion_r80787871) [](ancestors = 80787871)